### DISCUSSION TITLE
Android Theme Option Menu Background Opacity

### DISCUSSION THREAD

**Discussion by beppo-ivel** [appcelerator/titanium_mobile#13557] [Q&A] [answered]
How to set option menu popup background opacity? I tried color code "#0d020880". But this has no effect. 

```
<style name="menuBackground" parent="Widget.AppCompat.PopupMenu.Overflow">
    <item name="android:popupBackground">#0d020880</item>
    <item name="overlapAnchor">false</item>
    <item name="android:dropDownVerticalOffset">-4.0dip</item>
    <item name="android:overlapAnchor">false</item>
    <item name="android:dropDownVerticalOffset">4.0dip</item>
  </style>
```

**Comment by m1ga** [comment, marked as answer]
```xml
<item name="android:backgroundDimAmount">0.87</item>
```
https://developer.android.com/reference/android/R.attr#backgroundDimAmount

**Reply by beppo-ivel** [reply]
If I add this `item` to my code snippet  it has no effect. I also added `<item name="android:backgroundDimEnabled">true</item>` Do I have to use an other parent class? I also tried `<item name="android:opacity">fffffffe</item>` but it run into issues.

**Reply by m1ga** [reply]
can you please define what you mean by "option menu". I use this for an Opendialog and the background is darker. Check this full example here: https://fromzerotoapp.com/android-styling-in-titanium-using-themes/#link11

**Reply by beppo-ivel** [reply]
The popup that appears if you click on the 3 dots in the action bar (first image from your link).

**Reply by m1ga** [reply]
Ok, that was my first guess, but since there is no overlay that covers the screen I wasn't sure if you are talking about that. Or do you want to give the menu items a transparency?If that is the case:
```
    <style name="AppTheme" parent="Theme.Titanium.DayNight.Solid">
        <item name="actionOverflowMenuStyle">@style/OverflowMenuStyle</item>
    </style>
    <style name="OverflowMenuStyle" parent="Widget.MaterialComponents.PopupMenu.Overflow">
        <item name="android:popupBackground">#55FF0000</item>
    </style>
```

test app:
```js
var win = Ti.UI.createWindow({
	backgroundColor: "#fff",
	theme: "AppTheme"
});

var activity = win.activity;

activity.onCreateOptionsMenu = function(e) {
	var menu = e.menu;
	menu.add({
		title: "Item 1",
		showAsAction: Ti.Android.SHOW_AS_ACTION_NEVER
	});
	menu.add({
		title: "Item 2",
		showAsAction: Ti.Android.SHOW_AS_ACTION_NEVER
	});
};

win.open();
```
![Screenshot_20220903-190650](https://user-images.githubusercontent.com/4334997/188281190-cb18a8c1-90a2-423d-b570-6c89d62edc3f.png)


Btw: I don't know this stuff, I just google it :smile: This one is from https://stackoverflow.com/a/72918080/5193915 I've just changed it to a color instead of none. Check this part from my tutorial `And since Titanium and most Android apps use Material now the chances are high that old stackoverflow posts or examples won’t work anymore.`. So if you search native themes make sure those are Material results.
